blockprotocol/blockprotocol
# Allow a few more special characters in createSchemaModal

## 🌟 What is the purpose of this PR?

We sanitize the input when users create types, we were being a bit too restrictive and it'd definitely be good to allow apostrophes and brackets in titles.

## ❓ How to test this?

1. Run the site and try creating a type with a complicated title
2. Make sure the URL looks okay, things don't crash, etc.

## 📹 Demo

<img width="1603" alt="image" src="https://user-images.githubusercontent.com/25749103/221921385-b61fd294-e995-436a-afec-0dd6bbce18d8.png">
